### PR TITLE
Fixes for pcap2emulation

### PR DESCRIPTION
**Pull request by superm1**
These get the script at least closer to working with 
https://gitlab.com/asus-linux/reverse-engineering/-/blob/master/rog-ally-x/update-capture/update-312-to-312/ally-x-312_t01_update.pcap?ref_type=heads

but there still seems to be some other mismatches with how the endpoints have more HID descriptors than the script expects.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
Looks okay, but @fdanis-oss is probably who we need here.

File: contrib/pcap2emulation.py [hunk 1]
```diff
@@ -662,22 +667,33 @@ def parse_file(self, file: str) -> None:
...
+                            get_int(layers["usb_usb_bmRequestType_type"])
                         )
                         s += ",Recipient=0x{:02x}".format(
-                            int(layers["usb_usb_bmRequestType_recipient"], 16)
+                            get_int(layers["usb_usb_bmRequestType_recipient"])
                         )
                         s += f",Request=0x{int(layers['usb_usb_setup_bRequest']):02x}"
-                        s += f",Value=0x{int(layers['usb_usb_setup_wValue'], 16):04x}"
+                        s += f",Value=0x{int(layers['usb_usb_setup_wValue']):04x}"
                         s += f",Idx=0x{int(layers['usb_usb_setup_wIndex']):04x}"
```
Comment (fdanis-oss) [inline contrib/pcap2emulation.py:697]: Should those `int` be changed to `get_int`?

**Review by fdanis-oss — COMMENTED** [review]
I haven't work on it for a long time, but this looks good to me

File: contrib/pcap2emulation.py (hunk 1 above)
Comment (superm1) [reply contrib/pcap2emulation.py]: Yeah that makes sense, thanks adjusted.

**Comment by superm1** [comment]
> I haven't work on it for a long time, but this looks good to me

Thanks!  I'll merge with your feedback.

I'm pretty unfamiliar with this script.  If you have a few cycles as you know this a lot more in depth I would appreciate any further guidance to making it work with the linked dump; you might be able to do it quicker than me.

**Pull request merged**